CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+## Team members
+
+* [@geigerzaehler][gz]
+* [@rudolfs][ru]
+* [@brandonhaslegs][bhl]
+* [@juliendonck][jd]
+
+## Learn more about Upstream
+
+Looking to get familiar with the Upstream app and the Radicle ecosystem:
+
+* [DEVELOPMENT.md][dmd]
+* [Discord][dc]
+* [radicle.xyz][ra]
+* [Radicle documentation][rd]
+
+<!-- # Bug triage
+
+This sections explains how bug triaging is done for your project. Help beginners by including examples to good bug reports and providing them questions they should look to answer.
+
+* You can help report bugs by filing them here:
+* You can look through the existing bugs here:
+
+* You can help us diagnose and fix existing bugs by asking and providing answers for the following:
+
+  * Is the bug reproducible as explained?
+  * Is it reproducible in other environments (for instance, on different browsers or devices)?
+  * Are the steps to reproduce the bug clear? If not, can you describe how you might reproduce it?
+  * What tags should the bug have?
+  * Is this bug something you have run into? Would you appreciate it being looked into faster?
+
+* You can close fixed bugs by testing old tickets to see if they are still happening.
+* You can update our changelog here:
+* You can remove duplicate bug reports by: -->
+
+## Your first bugfix
+
+If you have an issue you know how to fix or see an [open issue][oi] you'd like
+to fix, we are keen to receive your contributions.
+
+We are currently accepting [PR's through GitHub][pr] while we make our own
+infrastructure more robust. If it's your first time using GitHub Pull Requests,
+learn more about it [here][ghf].
+
+We also accept contributions through [Upstream Patches][up], our first version
+of Pull Requests in Upstream. Learn more about the contribution flow [here][cbf].
+For us to see you've contributed a Patch we need to [add your Radicle ID as a
+remote][ar] to our repo, you can either add it to the [Remotes to track
+issue][rtr] or email it to [julien@monadic.xyz][mt].
```

Comment:
Let’s just ask them to mention their peer ID on discord. I think we should not encourage people to send emails to a single person to avoid creating a bottleneck.